pipeline/src/test/java/edu/illinois/cs/cogcomp/pipeline/main/CachingPipelineTest.java

```diff
@@ -162,12 +162,17 @@ public void stanfordParseHandler() {
             System.out.println("Expected exception from stanford.");
         }
         String predictedDepTree = basicTextAnnotation.getView(ViewNames.DEPENDENCY_STANFORD).toString();
-        String goldDepTree = "(taken (:LABEL:prep In (:LABEL:pobj States :LABEL:det the\n" +
-                "                    :LABEL:nn United))\n" +
-                "       (:LABEL:nsubj Cinco (:LABEL:prep de :LABEL:pobj Mayo))\n" +
+        String goldDepTree = "(taken :LABEL:punct .\n" +
+                "       (:LABEL:nmod States :LABEL:case In\n" +
+                "                           :LABEL:det the\n" +
+                "                           :LABEL:compound United)\n" +
+                "       :LABEL:punct ,\n" +
+                "       (:LABEL:nsubj Cinco (:LABEL:nmod Mayo :LABEL:case de))\n" +
                 "       :LABEL:aux has\n" +
-                "       (:LABEL:prep on (:LABEL:pobj significance :LABEL:det a))\n" +
-                "       (:LABEL:prep beyond (:LABEL:pobj that (:LABEL:prep in :LABEL:pobj Mexico))))";
+                "       (:LABEL:nmod significance :LABEL:case on\n" +
```

Comment:
Here, and for 'de' above, the dependency tree has the label "case" where it used to be "prep". This is apparently due to the shift to universal dependencies.  Changes like this may break (or at least render inoperative) a good number of feature extractors, so we should make it very clear to anyone upgrading from the old i-c-n to the new one.  Also, we will have to retrain SRL for the pipeline to work with this new version.